deploy/openshift/deploy_che.sh

```diff
@@ -207,22 +207,36 @@ isLoggedIn() {
   fi
 }
 
+getTemplates(){
+  if [ ! -d "${BASE_DIR}/templates" ]; then
+  printInfo "Local templates directory not found. Downloading templates..."
+  curl -s https://codeload.github.com/eclipse/che/tar.gz/master | tar -xz --strip=3 che-master/deploy/openshift/templates -C ${BASE_DIR}
+  OUT=$?
+  if [ ${OUT} -eq 1 ]; then
```

Comment:
It might be better to check for non zero return code as there are other error code other than 1